### PR TITLE
Xmlpull import + Mavenization

### PR DESCRIPTION
**Pull request by psiroky**

### CODE REVIEW

File: xmlpull/.gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
+bin
+build
+SCRATCH
```
Comment (ge0ffrey) [inline xmlpull/.gitignore]: (nitpick) do wee still need SCRATCH etc ignored?

Comment (psiroky) [reply xmlpull/.gitignore]: Likely not, this would be a leftover from the Ant build. Will check and remote those entries.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  <description>
+    General Pull parsing API + XPP3 parser implementation
+  </description>
+  <url>http://www.xmlpull.org/</url>
+  <inceptionYear>2002</inceptionYear>
+
+  <licenses>
+    <license>
+      <name>Public Domain</name>
+      <url>http://www.xmlpull.org/v1/download/unpacked/LICENSE.txt</url>
```
Comment (ge0ffrey) [inline pom.xml:22]: I presume this is the same as what's now in Maven Central for earlier versions?

Comment (psiroky) [reply pom.xml:22]: Yes, see e.g. https://repo1.maven.org/maven2/xmlpull/xmlpull/1.1.3.4d_b4_min/xmlpull-1.1.3.4d_b4_min.pom

Comment (psiroky) [reply pom.xml:22]: Not sure about the xpp3 license at this point, we may want to remove this entry and just and specific entries in the sub modules.

**Review by ge0ffrey — APPROVED** [review]

**Pull request merged**